feat: Add custom distributed tracing docs for JS/Node (#8469)
getsentry/sentry-docs: src/platform-includes/distributed-tracing/how-to-use/node.mdx

app.use(Sentry.Handlers.tracingHandler());
```

### Custom Instrumentation

> If you do not want to use performance monitoring, you can instead set up <PlatformLink to="/usage/distributed-tracing/custom-instrumentation/">Custom Instrumentation</PlatformLink> for distributed tracing.

Reviewer: ```suggestion
If you don't want to use performance monitoring, you can set up <PlatformLink to="/usage/distributed-tracing/custom-instrumentation/">Custom Instrumentation</PlatformLink> for distributed tracing.
```
